mailman/patchset.py

```diff
@@ -87,12 +85,15 @@ def do_git_am(extra_args=[]):
 def check(cover_letter, patches, submission_id):
     with tempfile.TemporaryDirectory() as repo_path:
         repo = git.Repo.init(repo_path)
+        with repo.config_writer() as config:
+            config.set_value('user', 'name', 'mailman')
+            config.set_value('user', 'email', 'mailman@mailman')
         status = do_check(repo, cover_letter, patches)
         if status[-1] == '!':
             for patch in patches:
                 patch_abspath = str(maildir / patch.msg_id)
-                repo.git.execute(['git', *author_args, 'commit', '--allow-empty', '-F', patch_abspath])
-        tag_and_push(repo_path, submission_id)
+                repo.git.execute(['git', 'commit', '--allow-empty', '-F', patch_abspath])
+        tag_and_push(repo, submission_id)
```

Comment:
change from `repo_path` to `repo` is in the wrong commit, should be in previous.